src/org/labkey/test/tests/UserClonePermissionTest.java

```diff
@@ -0,0 +1,107 @@
+package org.labkey.test.tests;
+
+import org.jetbrains.annotations.Nullable;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.pages.user.ClonePermissionsPage;
+import org.labkey.test.util.ApiPermissionsHelper;
+import org.labkey.test.util.PermissionsHelper;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Category({Daily.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 6)
+public class UserClonePermissionTest extends BaseWebDriverTest
+{
+    private static final String CLONED_SOURCE_SITE_USER = "sourcesiteuser@clonepermission.test";
+    private static final String CLONED_SOURCE_APP_USER = "sourceappuser@clonepermission.test";
+    private static final String CLONED_TARGET_SITE_USER = "targetsiteuser@clonepermission.test";
+    private static final String CLONED_TARGET_APP_USER = "targetappuser@clonepermission.test";
+    private static final String CLONE_GROUP = "Test clone group";
+
+    ApiPermissionsHelper _permissionsHelper = new ApiPermissionsHelper(this);
+
+    @Override
+    protected @Nullable String getProjectName()
+    {
+        return getClass().getSimpleName() + " Project";
+    }
```

Comment:
This should return `null` if the test doesn't create any projects.
```suggestion
    @Override
    protected @Nullable String getProjectName()
    {
        return null;
    }
```

Reply:
I will be using the project created.